source/manual/licensing.html.md

```diff
@@ -0,0 +1,111 @@
+- Feeds: backend (asynchronous) processing of application uploads and processing (via Apto), email notifications (via GOV.UK Notify), and expiry and purging of old applications. Also provides an interface for some debugging-type reporting.
+
+### Finding a licence
+
+Users use [Licence Finder](https://www.gov.uk/licence-finder) to locate licences on the GOV.UK frontend, but from a technical perspective this is separate from Licensify.
+
+Licence Finder sends users to a "Licence" page served by [Frontend](/apps/frontend.html). Users enter their postcode and Frontend uses [Mapit](/apps/mapit.html) to find their local authority. Frontend makes a request to a Licensify API to find out whether the licence is available on GOV.UK.
+
+### Applying for a licence
+
+Licence applications handled by Licensify are processed as follows:
+
+1. User downloads a PDF application form and completes it either in Adobe Reader or by hand.
+1. User uploads the completed application form to Licensify, and provides their email address and consents to a legal declaration.
+1. User is provided with receipt of application by email, and an email is sent to the relevant authority to inform them that an application is available for viewing.
+1. Licensing authorities check [Licensify's admin interface](https://licensify-admin.publishing.service.gov.uk/login) for new applications, then process them and either confirm or deny the application.
+
+## Testing
+
+There is an example licence that you can use to test Licensify. Note that the following instructions are safe to run any number of times in production:
+
+1. Go to <https://www.gov.uk/apply-for-a-licence/test-licence/gds-test/apply-1>
+1. Download the PDF application form
+1. Click 'Submit application'
+1. Enter your email address, upload the application form you just downloaded and tick the checkbox to accept the declaration
+1. Click 'Continue to pay fee'
+1. Click 'Continue to WorldPay website'
+1. Enter a [test card number](http://support.worldpay.com/support/kb/bg/testandgolive/tgl5103.html)
+1. Make payment
+1. You should see a confirmation page and receive an email with same details
+1. Login to Licensify's admin interface to verify that your application is visible
+
+You can give the application reference to Kibana in the relevant environment to see the logs and any error messages. The application should speak to both the frontend and backend app servers at least once during the transaction. It may take up to five minutes to process the licence application, so it might be worth waiting for a few minutes before checking Kibana.
+
+## Debugging
+
+The Feeds application has a UI that runs on port `9400`, but that isn't exposed publicly, so you need to tunnel to it. For example, the following will tunnel to the production instance of the Feeds admin and make it available via `localhost:9400`:
+
+```bash
+ssh licensing-backend-1.licensify.production -L 9400:localhost:9400 -N
```

Comment:
This doesn't work anymore, and involves the convoluted steps of:

```
gds govuk connect -e production ssh licensing_backend

Running command: ssh -J benthorner@jumpbox.production.govuk.digital benthorner@ip-10-13-5-62.eu-west-1.compute.internal

...

ssh benthorner@ip-10-13-5-62.eu-west-1.compute.internal -J benthorner@jumpbox.production.govuk.digital -L 9400:localhost:9400 -N
```

Reply:
That works - thanks!